playwright-e2e/tests/dsm/participant/participant-sending-sample-received-creates-germline-consent-addendum.spec.ts

```diff
@@ -97,10 +98,14 @@ test.describe.serial('Sending SAMPLE_RECEIVED event to DSS', () => {
       const tissue = await tissueInformationPage.tissue();
       await tissue.fillField(TissueDynamicFieldsEnum.USS, { inputValue: materialsReceivedAmount });
       smID = `SM-${crypto.randomUUID().toString().substring(0, 5)}`; //e.g. SM-12345
+      secondSMID = `SM-${crypto.randomUUID().toString().substring(0, 5)}`; //e.g. SM-09876
```

Comment:
`randomUUID()` might gives num with dashes `-`. 5 digits might be okay. Otherwise, make sure do a `replace('-`','') before `subString`.